specification/batch/Azure.Batch/client.tsp

```diff
@@ -19,10 +174,10 @@ using Azure.ClientGenerator.Core;
 );
 
 // Api overrides
-@@access(Azure.Batch.Batch.poolExists, Access.internal, "csharp");
-@@access(Azure.Batch.Batch.jobScheduleExists, Access.internal, "csharp");
-@@access(Azure.Batch.Batch.getTaskFileProperties, Access.internal, "csharp");
-@@access(Azure.Batch.Batch.getNodeFileProperties, Access.internal, "csharp");
+@@access(Azure.Batch.Pool.poolExists, Access.internal, "csharp");
```

Comment:
Seems like it would be more useful to group all these customizations together by language.